.travis.yml

```diff
@@ -15,7 +15,7 @@ jobs:
   include:
     - if: type = pull_request
       name: checkpatch
-      script: # FIXME: replace with TRAVIS_COMMIT_RANGE or similar
+      script: # FIXME: replace with TRAVIS_COMMIT_RANGE or similiar
```

Comment:
@marc-hb I I don't have comment on actual functionality, but I object to this spelling change :) My brain just emits a lot of warnings for "similiar" and Collins agrees:

https://www.collinsdictionary.com/dictionary/english/similar

Reply:
+1 on that, I'm not aware of any such word and "similar" is the correct form.

Reply:
> WARNING the second commit is a checkpatch test NOT meant to be merged.

:-)